flight/targets/flyingpio/board-info/board_hw_defs.c

```diff
@@ -1,16 +1,12 @@
 /**
  ******************************************************************************
- * @addtogroup TauLabsTargets Tau Labs Targets
- * @{
- * @addtogroup Naze32 family support files
- * @{
```

Comment:
You should still add these to a group, and subgroup, otherwise doxygen will become very unusable.